src/main/views/user-profiles/create-user-form.html

```diff
@@ -0,0 +1,136 @@
+{% extends "layout_template.html" %}
+{% from "../macros/formElements.html" import dropdown %}
+{% from "../macros/formElements.html" import dateMacro %}
+{% from "../macros/csrf.html" import csrfProtection %}
+{% block page_title %}
+  CCD Admin Web
+{% endblock %}
+{% block head%}
+  {{super()}}
+
+  <script type="text/javascript">
+  var jurisdictionString = {{jurisdictions |safe }}
+  var jurisdictionLookups = $.parseJSON(jurisdictionString);
+  
+  var updateCaseTypeBasedOn = function(index) {
+
+    var listItems= "<option value=''  selected>--Select a value--</option>";
+    if(index > -1){
+        var case_types = jurisdictionLookups[index].case_types;
+        for (var i = 0; i < jurisdictionLookups[index].case_types.length; i++){
+            listItems+= "<option value='" + case_types[i].id + "'>" + case_types[i].name + "</option>";
+        }
+    }
+    $("select#caseTypeDropdown").html(listItems);
+    $("select#stateDropdown").html("<option value=''  selected>--Select a value--</option>");
+  }
+
+  $(document).ready(function(){
+     $('#createuserform').validate({ // initialize the plugin
+        onkeyup: false,
+        onclick: false,
+        onfocusout: false,
+       errorPlacement: function (error, element) {
+            error.insertBefore(element);
+        },
+        errorElement : 'div',
+        errorElementClass: 'error',
+        errorClass: 'govuk-error-message form-group-error',
+        rules: {
+            idamsId: {
+                required: true,
+                email: true
+            },
+            jurisdictionDropdown: {
+                required: true
+            },
+            caseTypeDropdown: {
+                required: true
+            },
+            stateDropdown: {
+                required: true
+            }
+        },
+        messages :{
+            "idamsId" : {
+                required : 'Enter Idams Id',
```

Comment:
Please change "Idams" for "IdAM". The correct name for the user authentication system is IdAM.